composer.json

```diff
@@ -61,6 +61,7 @@
     },
     "conflict": {
         "doctrine/dbal": "<2.6.0",
+        "doctrine/mongodb-odm": "<2.0",
```

Comment:
how do we test the mongodb here? Maybe adding the mongodbadmin dependency? like we do with the orm

Reply:
mmm the only test I found is https://github.com/sonata-project/SonataMediaBundle/blob/315fe533dd4237243e6964efddaa314482970077/tests/Document/MediaManagerTest.php and there is no need of the library.